dotcom-rendering/src/components/marketing/banners/designableBanner/stories/DesignableBannerV2.stories.tsx

```diff
@@ -119,6 +119,7 @@ export const WithThreeTierChoiceCards: Story = {
 			abTestVariant: 'THREE_TIER_CHOICE_CARDS',
 		},
 		choiceCardAmounts: regularChoiceCardAmounts,
```

Comment:
Do we need these choiceCardAmounts?

Reply:
Good question. The V2 banner does not use this field.
However, the old banner does still use it, and because both banner components share the `BannerRenderProps` props type, we still need it in both! This is something we can tidy up when removing the old banner